extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesApplyReplicasDecorator.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.kubernetes.deployment;
+
+import io.dekorate.kubernetes.decorator.*;
```

Comment:
Please don't use "star" imports in Quarkus :)

Reply:
Other than this, I don't have any more objections and we should be good to go.

Reply:
Okey will fix this.

Reply:
Removed "star" imports